wiz-outpost-lite/templates/deployment.yaml

```diff
@@ -143,11 +160,22 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           securityContext:
-            capabilities:
-              add:
-              - SYS_ADMIN
-            seLinuxOptions:
-              type: spc_t
+            {{- /* Special handling for containerSecurityContext */}}
```

Comment:
This should be as simple as `{{ toYaml .Values.securityContext ... }}`, do what's necessary in calculating the values above to make it so